test/blackbox/common/BlackboxTestsDiscovery.cpp

```diff
@@ -228,6 +228,338 @@ TEST(Discovery, StaticDiscovery)
     reader.block_for_all();
 }
 
+TEST(Discovery, StaticDiscovery_v1_Reduced)
+{
+    char* value = nullptr;
```

Comment:
Part of these tests can be wrapped in auxiliary functions to prevent repeating the code once and again

Reply:
Done